src/App.js

```diff
@@ -1,13 +1,15 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
+import ListArticle from './Components/ListArticle'
 
 class App extends Component {
+  
   render() {
     return (
-      <div>
-        <h1>First scaffold</h1>
-      </div>
+      <>
+        <ListArticle />
```

Comment:
List one article? Or multiple articles?

Reply:
Changed to "ListArticles", to make the naming more appropriate for the intended purpose of the component.